### PR TITLE
Filter management

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestions use this live [link](https://deploy-preview-7--idyllic-klepon-23d3cf.netlify.app/)
Added filters management

### CODE REVIEW

File: src/components/ProductListing/ProductListing.css [hunk 1]
```diff
@@ -3,9 +3,10 @@
     width: 100%;
     height: 100%;
     display: flex;
+    justify-content: center;
     align-items: center;
     flex-wrap: wrap;
-    gap: 20px 15px;
+    gap: 20px 25px;
```
Comment (bipin7yadav) [inline src/components/ProductListing/ProductListing.css:9]: you may use rem instead of pixel

**Review by bipin7yadav — APPROVED** [review]
good to go

**Review by preethasri — APPROVED** [review]
good to merge

**Review by MiryalaAkshitha — APPROVED** [review]
Everything looks good

**Pull request merged**